### ISSUE TITLE
Chris: Hello Robert, how can I help? visitor_name ...

### ISSUE THREAD

**Issue by wormbasehelp** [WormBase/website#5064]
*Help Desk chat transcript. Issue can be closed if question was resolved in chat.*

Chris: Hello Robert, how can I help?</br>visitor_name (visitor_email): I am inserting a nucleotide sequence for BLAST and I keep getting a system error</br>Chris: Oh hmmm. Apologies, let me take a look</br>visitor_name (visitor_email): Error Message:
Caught exception in WormBase::Web::Controller::Tools-&gt;tools "read_file '/usr/local/wormbase/tmp/blast_blat/out_HG70o.out.err' - sysopen: No such file or directory at /var/lib/jenkins/jobs/staging_build/workspace/lib/WormBase/API/Service/blast_blat.pm line 312."</br>visitor_name (visitor_email): I have inserted the same sequence previously and I was able to get a BLAST read out</br>Chris: OK thanks for reporting the error</br>Chris: Our web team is looking into the problem now</br>Chris: I'll send you an email as soon as we have it back up and running</br>visitor_name (visitor_email): Is there something I am doing wrong or is it the program itself</br>Chris: No, it's not anything you're doing; there's a problem with the service</br>Chris: Apologies for the error, and thanks for reporting it</br>Chris: and thanks for your patience while we fix this</br>visitor_name (visitor_email): No problem</br>Chris: Is there anything else with which I can help you?</br>visitor_name (visitor_email): I am looking for Mos insertion sites in the genome, is there a simple way to gather this data</br>Chris: Hmm, let me look into it; there may be a file on our FTP site that will have that data</br>Chris: While I'm looking you may find this website useful:</br>Chris: http://www.wormbuilder.org/</br>visitor_name (visitor_email): I have been using that website as well.</br>Chris: OK</br>visitor_name (visitor_email): I am looking more or less for a list of Mos sites and what chromosome they are located on</br>Chris: OK, I'm taking a look at the current GFF file to see if we have Mos insertion sites annotated there</br>Chris: We have a track on our GBrowse genome browser called "Transposon insert sites" that may have what you need</br>Chris: They aren't particular to Mos, as far as I know</br>visitor_name (visitor_email): OK</br>visitor_name (visitor_email): is the blast working again</br>Chris: Not yet</br>Chris: Robert, I'll send you an email when BLAST is back up and once I've determined the best way to identify the Mos insertion sites

**Reported by:** robe** (robb********************)
**Submitted from:** <a target="_blank" href="http://www.wormbase.org/http://www.wormbase.org/tools/blast_blat/run">http://www.wormbase.org/tools/blast_blat/run</a>
**Browser:** Safari 9.1.2

**Comment by chris-grove** [comment]
Hi Robert,


 I've created two files for the Mos insertion sites:


1) I pulled out all lines from the current GFF file that refer to "transposable_element_insertion_site"

https://dl.dropboxusercontent.com/u/4290782/WormBase/Mos_insertion_site_query_8-30-2016/GFF_transposable_element_insertion_sites.txt


2) I've dumped out basic info for all Mos transposon insertion sites from our ACEDB database

https://dl.dropboxusercontent.com/u/4290782/WormBase/Mos_insertion_site_query_8-30-2016/WS255_Mos_insertion_sites.txt


You'll need to write a script to pull out all of the genomic coordinates for each Mos transposon from the GFF coordinates file, but with these two files at least you have a mapping of which insertion sites you want. The GFF data is listed by chromsome (first column) if you're only interested in sites along a specific chromosome.


Let me know if you have any questions about the data in those files.



As for the BLAST/BLAT service, our web team is still working to restore functionality. I will let you know when it is operational again.


Best,


Chris Grove

WormBase
